nights/2024-04-12-README.md

```diff
@@ -1,34 +1,33 @@
 # 2024-04-12
 
-## Reviewed by:   your_name_here
+## Reviewed by: Adre
 
 ## Raw image gallery: http://physics.mnstate.edu/feder_gallery/2024-04-12
 
 ## Unusual images?
 
 Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
 
-+ `crazy-object-001R.fit` -- not sure what this is an image of
-+ `another-crazy-object-003B.fit` -- this looks more like a flat than a science image.
-+ `full-moon-005B.fit` -- nice satellite or plane track in here.
++ `TIC_81247877-S001-R006-C002-rp.fit` -- satelLite 
++ `TIC_81247877-S001-R008-C001-rp.fit` -- SATELLITE
++ `TIC_81247877-S001-R008-C003-rp.fit` -- SATELLITE
++ `ey-uma-S001-R001-C001-B.fit` to `ey-uma-S001-R001-C007-rp.fit` -- messed up
++ `ey-uma-S001-R002-C011-rp.fit` --satellite
++ `ey-uma-S001-R003-C001-V.fit` -- satellite
++ `ey-uma-S001-R003-C003-rp.fit` -- satellite
++ `ey-uma-S001-R005-C013-rp.fit` -- satellite
++ `ey-uma-S001-R005-C016-rp.fit` -- half gradient
```

Comment:
This would be clouds and appears to affect `ey-uma-S001-R005-C016-rp.fit` through  `ey-uma-S001-R006-C002-rp.fit`